### ISSUE TITLE
Google hasn't verified this app

### ISSUE THREAD

**Issue by jimafisk** [calendso/calendso#203]
### Issue Summary

When you set up a new Integration with a Google calendar you get a warning that says "Google hasn't verified this app":

<details>
<summary>See screenshot</summary>

![google_verified_app](https://user-images.githubusercontent.com/5913244/118409253-779b2b80-b657-11eb-82e9-143345832c15.png)

</details> 

You can easily get around this by clicking "Advanced > Go to Calendso (unsafe)" but this might turn some folks off.

It [sounds like](https://stackoverflow.com/questions/45437557/this-app-isnt-verified-this-app-hasnt-been-verified-by-google-yet-only-procee) you can apply to be verified here: https://console.cloud.google.com/apis/credentials/consent. It's very possible that the Calendso team is already pursuing this.

### Steps to Reproduce
1. Create a new integration (https://app.calendso.com/integrations)
2. Choose Google Calendar
3. See the warning when selecting your Google account

**Comment by bennetfabian** [comment]
Not fully into Google's API but I think this is due to the fact that everyone who hosts Calendso on his own has to create an app in Google's cloud console. I think every user who wants to avoid this message has to individually apply for verification. That is because every user hosting Calendso is legally responsible for handling the data provided by Google's APIs like Google's ToS and the user's country's law requires it. Google wants to be on the safe side so it asks everyone wanting to use their authentication in production for a few things and wants to approve every app so they have less responsibility for huge data scandals like "Cambridge Analytica" etc.

**Comment by jimafisk** [comment]
That totally makes sense. I just wanted to flag it because it seems to be the case for the Calendso hosted app as well. The self-hosted folks might expect this, but I was worried it might scare off people who just want to pay for the service and use the managed solution.

**Comment by bennetfabian** [comment]
Oh oops, I thought it just applied to all the self-hosted instances, but if not, then it's very nice of you to report it so it doesn't result in a economic disadvantage.

**Comment by baileypumfleet** [comment]
Very confused as to why it's showing this, as we've already been through the verification process with Google and are currently marked as verified:
![Screenshot 2021-05-17 at 09 18 27](https://user-images.githubusercontent.com/25907159/118455366-f55f4580-b6f0-11eb-95af-b6c348e6d47b.png)

**Comment by baileypumfleet** [comment]
Just ran through this on my account and it didn't show that message. Can anyone else confirm if this is happening to them?

**Comment by baileypumfleet** [comment]
This has been escalated to Google API support

**Comment by jimafisk** [comment]
I tried removing my Google integration and adding it again and it gave an "Authorization Error":

<details>
<summary>Error screenshot</summary>

![autherror](https://user-images.githubusercontent.com/5913244/119741927-af268680-be54-11eb-83e8-ef92280fe08d.png)

</details>

I just used my browser's back button and tried again and it went through fine. Just thought I'd flag it here in case it's helpful at all. Thanks!
